cli/command/container/create.go

```diff
@@ -165,6 +166,9 @@ func createContainer(ctx context.Context, dockerCli command.Cli, containerConfig
 	networkingConfig := containerConfig.NetworkingConfig
 	stderr := dockerCli.Err()
 
+	warnOnOomKillDisable(*hostConfig, stderr)
+	warnOnLocalhostDNS(*hostConfig, stderr)
```

Comment:
FWIW, I actually want to move these warnings to the daemon (similar as to what I did in https://github.com/docker/cli/pull/1225 / https://github.com/moby/moby/pull/37502, that way the _daemon_ can decide if the container configuration has possible issues, and we can provide warnings for situations where (e.g.) memory-limit is not supported by the daemon, so will be ignored